src/main/java/com/thebois/models/world/World.java

```diff
@@ -1,26 +1,33 @@
 package com.thebois.models.world;
 
 import java.util.ArrayList;
+import java.util.List;
 
 import com.thebois.models.IFinder;
+import com.thebois.models.Position;
 import com.thebois.models.beings.Colony;
 import com.thebois.models.beings.IBeingGroup;
+import com.thebois.models.beings.roles.IRoleAllocator;
+import com.thebois.utils.MatrixUtils;
 
 /**
  * World creates a matrix and keeps track of all the structures and resources in the game world.
  */
 public class World implements IFinder {
 
-    private ITerrain[][] terrainMatrix;
-    private IBeingGroup colony;
+    private final ITerrain[][] terrainMatrix;
+    private final int beingCount;
```

Comment:
is this being count or pawn count?

Reply:
It used to be more general, to generate any amount of beings, but it's since changed. 

I'll update it to `pawnCount`.